albumentations/core/transforms_interface.py

```diff
@@ -11,11 +11,11 @@
 from .serialization import Serializable, get_shortest_class_fullname
 from .utils import format_args
 
-__all__ = ["to_tuple", "BasicTransform", "DualTransform", "ImageOnlyTransform", "NoOp"]
+__all__ = ["to_tuple", "BasicTransform", "DualTransform", "ImageOnlyTransform", "NoOp", "BoxType", "KeypointType"]
 
 NumType = Union[int, float, np.ndarray]
-BoxType = Tuple[float, float, float, float]
-KeypointType = Tuple[float, float, float, float]
+BoxType = Union[Tuple[float, float, float, float], Tuple[float, float, float, float, Any]]
+KeypointType = Union[Tuple[float, float, float, float], Tuple[float, float, float, float, Any]]
```

Comment:
Maybe to add additional type, for exapmle `KeypointAndLabels`?

Reply:
I was thinking about it. But it is rather complicated to use such Type.